repos/system_upgrade/el7toel8/actors/initramdiskgenerator/libraries/initramgen.py

```diff
@@ -68,9 +68,11 @@ def generate_initram_disk(context):
     copy_dracut_modules(context, modules)
     context.copy_to(generator_script, os.path.join('/', INITRAM_GEN_SCRIPT_NAME))
     install_initram_deps(context)
+    if os.path.exists('/etc/dasd.conf'):
```

Comment:
Isn't that required on every s390x? In such case, non-existing file could be an error on that arch.
```suggestion
    if architecture.matches_architecture(architecture.ARCH_S390X):
```

+ import of `architecture`. But this is ok as well.

Reply:
I just didnt add checks just yet

Reply:
I know. I just wanted to point that in case it could help later.